src/account_mt5.py

```diff
@@ -34,11 +35,11 @@ def get_account_profit(self) -> float:
         profit = account_info_dict['profit']
         return profit
     
-    def get_positions(self) -> dict:
+    def get_positions(self) -> tuple: # Note on 1.21.24 that MT5 returns positions as a named Tuple
         self.positions = mt5.positions_get()
         return self.positions
     
-    # Unsure if this is necessary - easier to work with positions as a dict rather than a DataFrame
+    # Unsure if this is necessary - easier to work with positions as a dict rather than a DataFrame # 1.21.24 - positions are actually Tuples
```

Comment:
What makes dicts easier to work with than dataframes?

Reply:
Stale comment, it is no longer applicable, will be removed

Reply:
I actually kept this comment as a reminder for myself but basically this addresses the following:
- Dataframes & Pandas itself is a dependency so in future iterations it would be nice to gradually remove it because based on my research the main advantage of using pandas is access to these technical analysis libraries and the more I experiment with the TA libraries I am finding that they all kind of only function half of the time so it is better to just make whatever TA your bot is trying to perform on your own
- For this particular thing positions, MetaTrader 5 returns it as a tuple in order to get into a dataframe you have to do a conversion similar to what we do with other MetaTrader 5 objects but you first have to read it in as a dict row by row and then convert it to a dataframe. The positions tuple is easier to access than the equivalent dataframe

Reply:
Sorry. What is TA here?

Reply:
I'm not sure what you mean by "they all kind of only function half of the time". These are well tested libraries that are heavily used by data scientists. They should be reliable.

But if it works with a dict, I don't really care either way.

Reply:
I think all I was trying to convey is I tried to implement a few of those TA libraries and of the ones I researched I could not find one that was capable of doing incremental calculations on moving averages, they could only do point in time calculations on an entire chunk of data. There were a few TA libraries that claimed they could do incremental calculations but they looked like they hadn't been updated for like 4-5 years. If you find one that looks reliable and is supported then its probably a good case for why pandas adds value. Right now, the TA lib part is more of a convenience factor to seed the initial block of data